qa/rpc-tests/test_framework/util.py

```diff
@@ -323,7 +323,7 @@ def initialize_datadir(dirname, n, bitcoinConfDict=None, wallet=None, bins=None)
     return datadir
 
 def rpc_auth_pair(n):
-    return 'rpcuser💻' + str(n), 'rpcpass🔑' + str(n)
+    return 'rpcuser' + str(n), 'rpcpass' + str(n)
```

Comment:
what was the point of this change? 

this was introduced to make sure that rpc interface could handle utf8 credential.

what was the problem you were experiencing  with the original code?

Reply:
I will revert.  I was having an unrelated issue connecting, saw what looked like binary garbage in util.py and so fixed.